### PR TITLE
Changes the way xp drops from Certus Ore blocks

### PR DESCRIPTION
**Pull request by Saereth**
Fixes https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/3478

### CODE REVIEW

File: src/main/java/appeng/decorative/solid/BlockQuartzOre.java [hunk 1]
```diff
@@ -76,16 +77,15 @@ public int quantityDropped( final Random rand )
 	}
 
 	@Override
-	public void dropBlockAsItemWithChance( final World w, final BlockPos pos, final IBlockState state, final float chance, final int fortune )
+	public int getExpDrop ( IBlockState state, IBlockAccess world, BlockPos pos, int fortune )
```
Comment (orod-org) [inline src/main/java/appeng/decorative/solid/BlockQuartzOre.java]: MAJOR '(' is preceded with whitespace. [rule](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.MethodParamPadCheck)

**Comment by orod-org** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by yueh** [comment]
The check is still strange in this case, if minecraft/forge actually provides a way to just generate the experience amount instead of dropping the items, I would assume we do not have to handle silk touch explicitly.

**Comment by fscan** [comment]
This should be fine, BlockOre does it the same way.

**Pull request merged**